.circleci/config.yml

```diff
@@ -33,10 +33,25 @@ version: 2.1
     paths:
       - vendor/bundle
 
+.create_test_app: &create_test_app
```

Comment:
I had to walk through this for a bit and in CI to double check but all makes sense. At first I had it wrong. Perhaps add a small note: "A test app is shared across Ruby versions but only for one Rails version."?

Reply:
I added a general comment about the workflow in the workflows section in https://github.com/activeadmin/activeadmin/pull/5534/commits/61a17ce92eebdbb438ad8860d92cda6886609ec4, let me know what you think! Not very good at doc writing, but I did include some bullet points to try make it easier to read :smiley:.

Reply:
That’s great thanks! 😄